apiserver/Dockerfile_proxy

```diff
@@ -0,0 +1,21 @@
+# Build security proxy
```

Comment:
can you rename this proxy.dockerfile, it would allow development tools (IDE) to recognize it as docker file.

Reply:
fixed